README.md

```diff
@@ -79,3 +79,42 @@ The following examples are provided:
 * [02 - Coloured Triangle](src/example_02/README.md)
 * [03 - Using an Index Buffer](src/example_03/README.md)
 * [04 - Rotating Cube](src/example_04/README.md)
+
+## Doc Site
+The documentation site is served by GitHub pages. The generate is Jekyll. There
+are current two parts to the docs site, the C and C++ API generator and the site
+itself.
+
+### C and C++ API Generator
+The generator for the C and C++ API is `scripts/api_doc_generator.rb`. This script
```

Comment:
Yet another language we need to learn?

Reply:
Yea, but this is a good language. Ruby is awesome.